eq_translations/extractable_strings.py

```diff
@@ -239,6 +239,81 @@
         "context": "Question",
         "additional_context": ["ListHeading", "ListDescription"],
     },
+    {
+        "json_path": "$..primary_content[*].title",
+        "description": "Introduction main title",
+    },
+    {
+        "json_path": "$..primary_content[*].contents[*].list[*]",
+        "description": "Introduction main list item",
+        "context": "PrimaryContent",
+        "additional_context": ["ListHeading", "ListDescription"],
+    },
+    {
+        "json_path": "$..primary_content[*].contents[*].description",
+        "description": "Introduction main description",
+        "context": "PrimaryContent",
+    },
+    {
+        "json_path": "$..primary_content[*].contents[*].guidance.contents[*].title",
+        "description": "Introduction main guidance title",
+        "context": "PrimaryContent",
+    },
+    {
+        "json_path": "$..primary_content[*].contents[*].guidance.contents[*].description",
+        "description": "Introduction main guidance description",
+        "context": "PrimaryContent",
+    },
+    {
+        "json_path": "$..preview_content.title",
+        "description": "Introduction preview title",
+    },
+    {
+        "json_path": "$..preview_content.contents[*].description",
+        "description": "Introduction preview description",
+        "context": "PreviewContent",
+    },
+    {
+        "json_path": "$..preview_content.questions[*].question",
+        "description": "Introduction preview question title",
+        "context": "PreviewContent",
+    },
+    {
+        "json_path": "$..preview_content.questions[*].contents[*].description",
+        "description": "Introduction preview question description",
+        "context": "PreviewContent",
+        "additional_context": ["PreviewQuestionListHeading"],
+    },
+    {
+        "json_path": "$..preview_content.questions[*].contents[*].list[*]",
+        "description": "Introduction preview question list item",
+        "context": "PreviewContent",
+        "additional_context": ["PreviewQuestionListHeading"],
+    },
+    {
+        "json_path": "$..secondary_content[*].contents[*].title",
+        "description": "Introduction additional title",
+    },
+    {
+        "json_path": "$..secondary_content[*].title",
```

Comment:
Secondary doesn't have a title.

Reply:
Removed, I thought that's what you meant when you said "they can be defined at the same level also".

Reply:
What I meant is:
```json
{
	"title": "Some text",
	"description": "Some description",
	"list": ["an item", "another line"]
},
```

Reply:
Ok, I get it now, contexts working as expected now. Thanks!